src/contracts/test/TestERC20USDC.sol

```diff
@@ -0,0 +1,36 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.7;
+
+import "@rari-capital/solmate/src/tokens/ERC20.sol";
+
+// Used for minting test ERC20s in our tests
+contract TestERC20USDC is ERC20("Test20USDC", "TST20", 28) {
```

Comment:
why does the token have 28 decimals? 😅 i thought 18 is the absolute maximum that's possible, curious what kind of undefined behaviour this may trigger :)

Reply:
my bad, it should have been 6.. Somewhere between local testing and pushing I changed it 🙈